Loop Simple Calculator.py

```diff
@@ -10,24 +10,25 @@ def simpcalc() :
 	num2 = float(input("Enter your second number: "))
 
 	if op == "+" :
-		addition = num1 + num2
-		print(addition)
+		result = num1 + num2
 		
 	elif op == "-" :
-		subtraction = num1 - num2
-		print(subtraction) 
+		result = num1 - num2
 		
 	elif op == "/" :
 		if num2 != 0 :
-			division = num1 / num2
-			print(division)
+			result = num1 / num2
+		else:
+			result = "Can't divide by 0"
 	
 	elif op == "*" :
-		multiplication = num1 * num2
-		print (multiplication) 
+		result = num1 * num2
 		
 	else:
-		print("Invalid operator")
+		result = "Invalid operator"
+
+	print(result)
```

Comment:
Instead of writing print statement for each operator, one can just calculate different values of some variable (in this case named "result") depending on operation, and then use 1 print statement after the if-else statement.

This would reduce the number of lines of code(makes code shorter) and, if you will want to change "print" to some advanced routine, you will only need to do it in one place - so, you can't make a mistake of changing it one place and not changing in other.